app/src/main/java/dev/msfjarvis/aps/ui/theme/Color.kt

```diff
@@ -0,0 +1,8 @@
+package dev.msfjarvis.aps.ui.theme
+
+import androidx.compose.ui.graphics.Color
+
+val Purple200 = Color(0xFFBB86FC)
+val Purple500 = Color(0xFF6200EE)
+val Purple700 = Color(0xFF3700B3)
+val Teal200 = Color(0xFF03DAC5)
```

Comment:
Get rid of all these files, they're not adding any value.

Reply:
I specially added them so that we can use them as a template and modify the files to match our color scheme, shapes, etc. I'll remove it we don't want it in the app.

Reply:
We're gonna use the material theme adapter for the initial migration so this is going to stay unused for a good while, there's no point keeping templates around for that long.